media-sound/teamspeak-server-bin/files/ts3db_mariadb.ini.sample

```diff
@@ -0,0 +1,7 @@
+[config]
+host=localhost
+port=3306
+username=teamspeak
+password=teamspeak
+database=teamspeak
+socket=
```

Comment:
Please add the trailing newline here.

Reply:
I've added it now. What's the purpose?

Reply:
That's the standard for regular files. Missing it just causes unnecessary noise from git.

Reply:
Thanks, will have a look on it. I see the red symbol, you mean, in Github.